### PR TITLE
etcher: 1.5.60 -> 1.5.86

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

Version update.

This also bumps dependency from very old Electron 3 to relatively new Electron 7.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by prusnak** [comment]
@GrahamcOfBorg build etcher

File: pkgs/tools/misc/etcher/default.nix [hunk 1]
```diff
@@ -55,33 +45,33 @@ in stdenv.mkDerivation rec {
...
+
+    mkdir -p $out/bin $out/share/${pname}
+
+    cp -a usr/share/* $out/share
+    cp -a opt/balenaEtcher/{locales,resources} $out/share/${pname}
 
-    # We'll use our Nixpkgs electron_3 instead
-    rm $out/opt/balenaEtcher/balena-etcher-electron
+    substituteInPlace $out/share/applications/balena-etcher-electron.desktop \
+      --replace 'Exec=/opt/balenaEtcher/balena-etcher-electron' 'Exec=${pname}'
```
Comment (Shou) [inline pkgs/tools/misc/etcher/default.nix:66]: Isn't an absolute path, i.e. `$out/bin/etcher` safer here?

Comment (prusnak) [reply pkgs/tools/misc/etcher/default.nix:66]: Maybe, but the vast majority of desktop files on my system have only relative names in them:

`find -name "*.desktop" -type f -exec grep Exec {} \;`

Not sure what is the policy in nixpkgs.

**Review by Shou — APPROVED** [review]
LGTM

**Pull request merged**